configure.ac

```diff
@@ -872,6 +872,96 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_RAW_LINK_API], [test "${enable_raw_link_api}"
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_RAW_LINK_API],[${OPENTHREAD_ENABLE_RAW_LINK_API}],[Define to 1 if you want to enable raw link-layer API])
 
 #
+# Software ACK Timeout Logic
+#
+
+AC_ARG_ENABLE(sw_ack_timeout,
```

Comment:
The reason I put them in here is because it made configuring the makefiles for each platform nice, clean and explicit, in my opinion.